examples/pip_requirements/pip_requirements.py

```diff
@@ -44,20 +44,20 @@ def main():
         artifact_path = "default"
         mlflow.xgboost.log_model(model, artifact_path)
         pip_reqs = get_pip_requirements(run_id, artifact_path)
-        assert pip_reqs == ["mlflow", xgb_req], pip_reqs
+        assert pip_reqs.issuperset(["mlflow", xgb_req]), pip_reqs
 
         # Overwrite the default set of pip requirements using `pip_requirements`
         artifact_path = "pip_requirements"
         mlflow.xgboost.log_model(model, artifact_path, pip_requirements=[sklearn_req])
         pip_reqs = get_pip_requirements(run_id, artifact_path)
-        assert pip_reqs == ["mlflow", sklearn_req], pip_reqs
+        assert pip_reqs == {"mlflow", sklearn_req}, pip_reqs
 
         # Add extra pip requirements on top of the default set of pip requirements
         # using `extra_pip_requirements`
         artifact_path = "extra_pip_requirements"
         mlflow.xgboost.log_model(model, artifact_path, extra_pip_requirements=[sklearn_req])
         pip_reqs = get_pip_requirements(run_id, artifact_path)
-        assert pip_reqs == ["mlflow", xgb_req, sklearn_req], pip_reqs
+        assert pip_reqs.issuperset(["mlflow", xgb_req, sklearn_req]), pip_reqs
```

Comment:
Rpelace `==` with `issuperset` since `pip_reqs` may not be equal to `["mlflow", xgb_req, sklearn_req]`.